corehq/apps/linked_domain/remote_accessors.py

```diff
@@ -65,27 +65,7 @@ def _format_missing_media(media):
 
         still_missing_media = _get_missing_multimedia(app)
         if still_missing_media:
-            soft_assert(to='{}@{}'.format('mkangia', 'dimagi.com'))(
```

Comment:
I left the retry and the `return False` logic here but am also noting that this assert has never gone off.

Reply:
that return value is not being used. We can either raise an exception there if this return False?

Reply:
good point - 9e3bcceeb19

Reply:
Cool.